### PR TITLE
Corrects some typos and tildes

### PR DESCRIPTION
**Pull request by lulol**
Corrects some typos and tildes since the latest update. :smile:

### CODE REVIEW

**Comment by justinclift** [comment]
Awesome @lulol!

Are these ok to merge as is, or should we take screenshots of them in action first, or ? :smile:

File: src/translations/sqlb_es_ES.ts [hunk 1]
```diff
@@ -2333,7 +2333,7 @@ Do you want to insert it anyway?</source>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1891"/>
-        <source>Copy the CREATE statement of the item to the clipboard</source>
+        <source>Copy the CREATE statement of the ítem to the clipboard</source>
```
Comment (MKleusberg) [inline src/translations/sqlb_es_ES.ts]: Not sure here because this changes the source string (= the original English text), not the Spanish one.

Comment (lulol) [inline src/translations/sqlb_es_ES.ts]: Ooops. :cry: Fixed.

Now think that are ok to merge as is.

Although for some couldn't find the dialogs where are actually being used, these are just fixed typos. All of them hopefully.

But in the GUI I'm still seeing [those same untranslated strings](https://github.com/sqlitebrowser/sqlitebrowser/pull/723#issuecomment-242979348) that seemed to be translated in the .ts like  that in line="14" but the others are not even defined there.

portapapeles01
portapapeles02

Comment (justinclift) [inline src/translations/sqlb_es_ES.ts]: @lulol Interesting.  Some of those I was able to find using grep:

### Main window

* [Edit Database Cell](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/MainWindow.ui#L951)
* [DB Schema](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/MainWindow.ui#L1338)

### Preferences pialog
* [Content](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/PreferencesDialog.ui#L362)
* [Symbol limit in cell](https://github.com/sqlitebrowser/sqlitebrowser/blob/bd0b8637e03c8cbf5628354f151b42cef083254e/src/PreferencesDialog.ui#L368)

Does that help? :smile:

**Comment by MKleusberg** [comment]
Thanks, @lulol :+1: I think, in order to have those missing strings included in the translation file, you'll need to re-run Qt Linguist on the application. Unfortunately I have no experience with that whatsoever...

**Comment by justinclift** [comment]
Ahhh, I can do that and commit the results if it would help. @lulol ?

**Comment by justinclift** [comment]
@MKleusberg It can be done from Qt Creator.  Tools → External → Linguist → Update translations (lupdate)

**Comment by lulol** [comment]
@justinclift We could try and see how it works out in the next nightly build.

**Comment by justinclift** [comment]
@lulol Good thinking. :smile:

Just committed an updated Spanish .ts file (generated from `lupdate`): https://github.com/sqlitebrowser/sqlitebrowser/commit/e4fdb8f05dcd412529bebda2b52cbd727af93fd3

Take a look at it and see if it seems correct to you.  If it is, we can commit an update for the other .ts files as well. :smile:

**Comment by lulol** [comment]
@justinclift I see that now among the 55 strings tagged "unfinished" are those that previously were missing in the .ts.

Should I translate them now? So that we can check before changing them for all the languages, if in the next nightly build also the other strings already translated but not showing in the GUI like "Edit Database Cell" and "DB Schema" are also fixed.

**Comment by justinclift** [comment]
Sure, if you want. :smile:

**Comment by lulol** [comment]
@justinclift Done, and [PR ready](https://github.com/sqlitebrowser/sqlitebrowser/pull/928). Hope that it works and the new generated files finally fix the missing translations for all the languages. :smile:

**Comment by justinclift** [comment]
Excellent @lulol, that was very quick!  I needed to get sleep, so missed getting the PR merged right away (now done).

**Pull request merged**